backend/tests/clubs/test_ticketing.py

```diff
@@ -1312,6 +1313,42 @@ def test_transfer_ticket_to_self(self):
         )
         self.assertEqual(resp.status_code, 403, resp.content)
 
+    def test_update_attendance(self):
+        self.client.login(username=self.user1.username, password="test")
+        Membership.objects.create(
+            person=self.user1,
+            club=self.club1,
+            title="Officer",
+            role=Membership.ROLE_OFFICER,
+        )
+        ticket = self.tickets1[0]
+        ticket.owner = self.user2
+        ticket.save()
+
+        resp = self.client.patch(
+            reverse("tickets-detail", args=(ticket.id,)),
+            {"attended": True},
+            format="json",
+        )
+        ticket.refresh_from_db()
+        self.assertEqual(resp.status_code, 200, resp.content)
+        self.assertTrue(ticket.attended)
+
+    def test_update_attendance_non_officer(self):
+        self.client.login(username=self.user1.username, password="test")
+        ticket = self.tickets1[0]
+        ticket.owner = self.user1
+        ticket.save()
```

Comment:
I got a little confused for a second why a difference in ticket ownership mattered, but it doesn't. In `test_update_attendance` you create a membership between `user1` and `club1` such that `user1` has officer role. In this function, no membership is given and clubs don't have any membership relations with any user from `comonSetUp`, so here `user1` is not an officer. I think having an explicit comment would prevent future confusion.